lib/internal/test_runner/runner.js

```diff
@@ -561,14 +564,16 @@ function run(options = kEmptyObject) {
     });
   }
 
+  const runCwd = cwd ?? process.cwd();
```

Comment:
Please handle this new option the same way as the other options.

Reply:
Hey @cjihrig, no problem at all, I'm going to update the doc while fixing the conflicts!!!

Reply:
@cjihrig only one question: how should I handle the default value?
Is it ok the way I did (line 567) ?

Reply:
On line 488 where you destructure the `cwd` out of the object, you can do `cwd = process.cwd()`. Then use `validateString()` for the validation.

(Not something for this PR, but we should probably do something similar for the other options, which don't have a default value applied. We also need to support more options in `run()` to get parity with the CLI. I think that will be simpler after #54353 lands.)

Reply:
Hey @cjihrig, I've just pushed a commit containing both the documentation and the new option handling.  
Could you please give me feedback on the documentation?  
I tried to be as generic as possible because I think that cwd shouldn't be strictly related to the changes contained in this PR, hope that's fine